Tests/SwiftLintFrameworkTests/ImplicitlyUnwrappedOptionalRuleTests.swift

```diff
@@ -23,4 +23,28 @@ class ImplicitlyUnwrappedOptionalRuleTests: SwiftLintTestCase {
         verifyRule(description, ruleConfiguration: ["mode": "all"],
                    commentDoesntViolate: true, stringDoesntViolate: true)
     }
+
+    func testImplicitlyUnwrappedOptionalRuleWarnsOnOutletsInWeakMode() {
+        let baseDescription = ImplicitlyUnwrappedOptionalRule.description
+        let triggeringExamples = [
+            Example("private weak var label: UILabel!"),
+            Example("weak var label: UILabel!"),
+            Example("@objc weak var label: UILabel!")
+        ]
+
+        let nonTriggeringExamples = [
+            Example("if !boolean {}"),
```

Comment:
This test case can be removed. It doesn't have any value now that the rule is based on SwiftSyntax.